sdk/src/main/java/io/radar/sdk/RadarApiClient.kt

```diff
@@ -155,6 +155,16 @@ internal class RadarApiClient(
             val mocked = location.isFromMockProvider
             params.putOpt("mocked", mocked)
         }
+        val tripOptions = RadarSettings.getTripOptions(context)
+        if (tripOptions != null) {
+            params.putOpt("tripExternalId", tripOptions.externalId)
+            params.putOpt("tripMetadata", tripOptions.metadata)
+            if (tripOptions.destinationGeofenceTag != null) {
+                params.putOpt("tripDestinationGeofenceTag", tripOptions.destinationGeofenceTag)
```

Comment:
nit: use `mapOf` here? having the `map` literal seems a bit cleaner - https://kotlinlang.org/docs/reference/idioms.html#read-only-map